measCompApp/Db/measCompLongOut.template

```diff
@@ -3,15 +3,15 @@
 ###################################################################
 record(longout, "$(P)$(R)")
 {
-    field(PINI, "YES")
+#   field(PINI, "YES")
```

Comment:
Change this to $(DESC)= so it is optional.

Reply:
ok